core/trino-main/src/main/java/io/trino/execution/buffer/OutputBufferMemoryManager.java

```diff
@@ -110,6 +121,9 @@ public void updateMemoryUsage(long bytesAdded)
                     this.bufferBlockedFuture = null;
                 }
             }
+            long recordTime = ticker.read();
+            outputBufferUtilization.add(getUtilization(), (double) (recordTime - this.lastBufferUtilizationRecordTime));
```

Comment:
`outputBufferUtilization` should be updated with previous utilization (from `this.lastBufferUtilizationRecordTime` to `recordTime`) rather than a new one.